lib/cli/src/commands/create_exe.rs

```diff
@@ -419,32 +418,102 @@ impl CreateExe {
                         let _ = std::fs::create_dir_all(&target_file_path);
                         let files = untar(local_tarball.clone(), target_file_path.clone())?;
                         tarball_dir = target_file_path.canonicalize().unwrap_or(target_file_path);
-                        files.iter().find(|f| f.contains(libwasmer_path)).cloned().ok_or_else(|| {
-                            anyhow!("Could not find libwasmer for {} target in the provided tarball path (files = {files:#?}, libwasmer_path = {libwasmer_path:?})", target)})?
+                        files.iter().find(|f| f.ends_with("libwasmer.a")).cloned().ok_or_else(|| {
+                            anyhow!("Could not find libwasmer.a for {} target in the provided tarball path (files = {files:#?})", target)})?
```

Comment:
You can use `anyhow::bail!()` to return an error.

```suggestion
                            anyhow::bail!("Could not find libwasmer.a for {} target in the provided tarball path (files = {files:#?})", target)});
```

Also, how many files does a tarball normally have? If it's more than a couple, we're going to be spamming the user's console. Would it be worth emitting some log statements intead?

Reply:
In general, no, it's just 10 - 15 files, it's not spamming plus it shouldn't happen anyway. In this case the println helped me to find the bug faster, so I'd like to keep it as is.